src/HostManager.cpp

```diff
@@ -38,39 +38,34 @@
 #include "post_guard.h"
 
 
-bool HostManager::deleteHost( QString hostname )
+bool HostManager::deleteHost(QString hostname)
 {
     qDebug() << "HostManager::deleteHost(" << hostname.toUtf8().constData() << ") INFO: trying to delete host from host pool, getting write lock...";
```

Comment:
I'd rather not have commented out code. I'm not as familiar with Qt debugging, but this is what log levels are for. Perhaps we should see if Qt supports them.